static/js/public/fullRecord.js

```diff
@@ -103,8 +103,8 @@ define('fullRecord', ['module', 'jquery', 'JP2Viewer', 'StructureView', 'dataTab
 
 					img += '<div class="thumbnail-badge thumbnail-badge-' + whichBadge + '">' +
 							'<div class="fa-stack">' +
-								'<i class="fas fa-circle fa-stack-2x background"></i>' +
-								'<i class="fas fa-' + whichBadge + ' fa-stack-1x foreground"></i>' +
+								'<i class="fas fa-circle fa-stack-2x background"></i></i>' +
```

Comment:
Looks like this got double closed? Same with the next line

Reply:
Updated.